pages/index.tsx

```diff
@@ -2,14 +2,21 @@ import { Signer, TypedDataSigner } from '@ethersproject/abstract-signer'
 import { useCreateOffer } from '@nft/hooks'
 import { BigNumber } from 'ethers'
 import { useCallback } from 'react'
-import { useSigner } from 'wagmi'
+import { chain, useAccount, useConnect, useDisconnect, useSigner } from 'wagmi'
+import { InjectedConnector } from 'wagmi/connectors/injected'
 import styles from '../styles/app.module.css'
 
 export default function Home() {
   const { data: signer } = useSigner()
   const [_create] = useCreateOffer(
     signer as (Signer & TypedDataSigner) | undefined,
   )
+  const { address, isConnected } = useAccount()
+  const { connect } = useConnect({
+    connector: new InjectedConnector(),
+    chainId: chain[process.env.NEXT_PUBLIC_CHAIN_NAME].chainId, // Polygon Mumbai
```

Comment:
Wrong param and wrong comment:
```suggestion
    chainId: chain[process.env.NEXT_PUBLIC_CHAIN_NAME].id, // Pass the name of the Wagmi supported chain. See "chain" types or (https://wagmi.sh/docs/providers/configuring-chains#chains)
```